### ISSUE TITLE
Release stack-augmented network architecture.

### ISSUE THREAD

**Issue by adrelino** [Hippogriff/CSGNet#5]
Will you release the updated network from your new arxiv submission https://arxiv.org/abs/1912.11393?

> This paper extends our work that first appeared in [6], adding analysis on reward shaping and the dependence on training set size, as well as the stack-augmented network architecture. Our PyTorch [7] implementation is publicly available at: https://hippogriff.github.io/CSGNet/.

**Comment by Hippogriff** [comment]
Hi! Thank you for your interest. The implementation of the new architecture will be released after the paper has been accepted.
